crates/exchange-rates/src/feed.rs

```diff
@@ -0,0 +1,8 @@
+use ethers::types::Address;
+use serde::Deserialize;
+
+#[derive(Debug, Deserialize)]
+#[serde(tag = "provider", rename_all = "camelCase", content = "address")]
```

Comment:
hmm, didn't even know about the `content` attr, but seems exactly what we want. nice one